README.md

```diff
@@ -738,6 +739,23 @@ $ git tag -d <tag_name>
 $ git push <remote> :refs/tags/<tag_name>
 ```
 
+<a name="recover-tag"></a>
+## Recover a deleted tag
+
+If you want to recover a tag that was already deleted, you can do so by following these steps: First, you need to find the unreachable tag:
+
+```sh
+$ git fsck --unreachable | grep tag
+```
+
+Make a note of the tag's hash. Then, restore the deleted tag with following, making use of git's [update-ref](http://git-scm.com/docs/git-update-ref):
+
+```sh
+git update-ref refs/tags/<tag_name> <hash>
```

Comment:
Missing a `$`. This should conform to the rest of the document. using `<hash>` is smart, we should probably do that instead of randomly using a hash like we've been doing.